CommunityFridgeMapApi/dependencies/python/db.py

```diff
@@ -88,60 +151,191 @@ def format_dynamodb_item(self):
                 fridge_item[key] = {self.ITEM_TYPES[key]: val}
         return fridge_item
 
+    def get_class_field_value(self, key: str):
```

Comment:
Is ever possible for the key to have further levels? For example, something like `"loc/attr1/attr2"`?

Reply:
Yeah it's possible. Our current data model doesn't have an example of this, but in the future it might. 

I take into account multiple levels in line 173. That loop goes down as many levels it needs to